src/emmc.c

```diff
@@ -186,15 +187,19 @@ pu_emmc_setup_layout(PuFlash *flash,
             part->size = self->expanded_part_size;
         }
 
+        if (part->align > 0) {
+            part->size -= part->size % part->align;
+        }
+
         /* Increase size of logical partitions by 2 sectors for EBR.
          * The 2 sectors are removed during partition creation. */
         if (part->type == PED_PARTITION_LOGICAL) {
             part->size += 2;
         }
 
-        g_debug("%s: type=%d filesystem=%s start=%lld size=%lld offset=%lld expand=%s",
+        g_debug("%s: type=%d filesystem=%s start=%lld size=%lld offset=%lld align=%lld expand=%s",
```

Comment:
I think it is time to break this string into two lines. The soft limit of 80 characters has been stretched enough. :)